bom/application/pom.xml

```diff
@@ -209,6 +209,7 @@
         <log4j-jboss-logmanager.version>1.2.0.Final</log4j-jboss-logmanager.version>
         <avro.version>1.10.0</avro.version>
         <jacoco.version>0.8.6</jacoco.version>
+        <testcontainers.version>1.15.1</testcontainers.version>
```

Comment:
```suggestion
        <testcontainers.version>1.15.2</testcontainers.version>
```

Reply:
This is to comply with https://github.com/quarkusio/quarkus/blob/master/bom/test/pom.xml#L20